<?xml version='1.0' encoding='UTF-8'?>
<root>
 <tabbed_widget parent="main_window" name="Main Window">
  <plotmatrix columns="1" rows="4" tab_name="plot">
   <plot col="0" row="0">
    <range top="1.975545" right="99.021248" left="0.054904" bottom="-1.057681"/>
    <limitY/>
    <curve name="/estimate_error/velocityError/x" B="255" custom_transform="" R="0" G="0"/>
    <curve name="/estimate_error/velocitySDMinus/x" B="0" custom_transform="" R="255" G="19"/>
    <curve name="/estimate_error/velocitySDPlus/x" B="20" custom_transform="" R="246" G="0"/>
    <transform value=""/>
   </plot>
   <plot col="0" row="1">
    <range top="2.817139" right="99.021248" left="0.054904" bottom="-1.658187"/>
    <limitY/>
    <curve name="/estimate_error/velocityError/y" B="255" custom_transform="" R="0" G="61"/>
    <curve name="/estimate_error/velocitySDMinus/y" B="69" custom_transform="" R="255" G="4"/>
    <curve name="/estimate_error/velocitySDPlus/y" B="25" custom_transform="" R="255" G="0"/>
    <transform value=""/>
   </plot>
   <plot col="0" row="2">
    <range top="3.150587" right="99.021248" left="0.054904" bottom="-3.150587"/>
    <limitY/>
    <curve name="/estimate_error/velocityError/z" B="255" custom_transform="" R="21" G="0"/>
    <curve name="/estimate_error/velocitySDMinus/z" B="40" custom_transform="" R="253" G="0"/>
    <curve name="/estimate_error/velocitySDPlus/z" B="0" custom_transform="" R="255" G="0"/>
    <transform value=""/>
   </plot>
   <plot col="0" row="3">
    <range top="0.499491" right="99.021248" left="0.054904" bottom="-0.499491"/>
    <limitY/>
    <curve name="/estimate_error/zError" B="255" custom_transform="" R="0" G="0"/>
    <curve name="/estimate_error/zSDMinus" B="0" custom_transform="" R="255" G="8"/>
    <curve name="/estimate_error/zSDPlus" B="0" custom_transform="" R="255" G="0"/>
    <transform value=""/>
   </plot>
  </plotmatrix>
  <plotmatrix columns="1" rows="4" tab_name="plot">
   <plot col="0" row="0">
    <range top="2.211245" right="22.897266" left="5.221865" bottom="-1.348996"/>
    <limitY min="-3" max="3"/>
    <curve name="/rgbd_velocity/body_level_frame/S_lower_bound.0" B="255" custom_transform="" R="0" G="233"/>
    <curve name="/rgbd_velocity/body_level_frame/S_upper_bound.0" B="255" custom_transform="" R="0" G="217"/>
    <curve name="/rgbd_velocity/body_level_frame/vel/twist/twist/linear/x" B="255" custom_transform="" R="0" G="2"/>
    <curve name="/velocity/body_level_frame/twist/twist/linear/x" B="0" custom_transform="" R="0" G="124"/>
    <curve name="/xyz_debug_estimate/xy_plus/sigma_minus.0" B="0" custom_transform="" R="255" G="173"/>
    <curve name="/xyz_debug_estimate/xy_plus/sigma_plus.0" B="0" custom_transform="" R="255" G="152"/>
    <curve name="/xyz_debug_estimate/xy_plus/x_dot" B="0" custom_transform="" R="255" G="1"/>
    <transform value=""/>
   </plot>
   <plot col="0" row="1">
    <range top="3.000000" right="22.897266" left="5.221865" bottom="-3.000000"/>
    <limitY min="-3" max="3"/>
    <curve name="/rgbd_velocity/body_level_frame/S_lower_bound.1" B="255" custom_transform="" R="0" G="234"/>
    <curve name="/rgbd_velocity/body_level_frame/S_upper_bound.1" B="255" custom_transform="" R="0" G="210"/>
    <curve name="/rgbd_velocity/body_level_frame/vel/twist/twist/linear/y" B="255" custom_transform="" R="11" G="0"/>
    <curve name="/velocity/body_level_frame/twist/twist/linear/y" B="23" custom_transform="" R="0" G="255"/>
    <curve name="/xyz_debug_estimate/xy_plus/sigma_minus.1" B="0" custom_transform="" R="255" G="155"/>
    <curve name="/xyz_debug_estimate/xy_plus/sigma_plus.1" B="0" custom_transform="" R="255" G="167"/>
    <curve name="/xyz_debug_estimate/xy_plus/y_dot" B="0" custom_transform="" R="255" G="22"/>
    <transform value=""/>
   </plot>
   <plot col="0" row="2">
    <range top="0.724007" right="22.897266" left="5.221865" bottom="-0.968806"/>
    <limitY/>
    <curve name="/sync_estimates/estimatedVelocity/z" B="0" custom_transform="" R="255" G="0"/>
    <curve name="/sync_estimates/trueVelocity/z" B="0" custom_transform="" R="42" G="156"/>
    <curve name="/sync_estimates/velocity_sigma_minus/z" B="255" custom_transform="" R="0" G="22"/>
    <curve name="/sync_estimates/velocity_sigma_plus/z" B="255" custom_transform="" R="0" G="1"/>
    <transform value=""/>
   </plot>
   <plot col="0" row="3">
    <range top="0.500000" right="22.897266" left="5.221865" bottom="-3.000000"/>
    <limitY min="-3" max="0.5"/>
    <curve name="/sync_estimates/zEstimate" B="0" custom_transform="" R="255" G="15"/>
    <curve name="/sync_estimates/zMeasured" B="255" custom_transform="" R="1" G="1"/>
    <curve name="/sync_estimates/zTrue" B="0" custom_transform="" R="22" G="128"/>
    <curve name="/sync_estimates/z_sigma_minus" B="255" custom_transform="" R="0" G="0"/>
    <curve name="/sync_estimates/z_sigma_plus" B="255" custom_transform="" R="16" G="0"/>
    <transform value=""/>
   </plot>
  </plotmatrix>
  <plotmatrix columns="1" rows="4" tab_name="plot">
   <plot col="0" row="0">
    <range top="1.006999" right="99.000680" left="0.003578" bottom="-0.996413"/>
    <limitY/>
    <curve name="/xyz_debug_estimate/xy_plus/pitch_bias" B="255" custom_transform="" R="0" G="30"/>
    <curve name="/xyz_debug_estimate/xy_plus/sigma_minus.2" B="0" custom_transform="" R="255" G="0"/>
    <curve name="/xyz_debug_estimate/xy_plus/sigma_plus.2" B="0" custom_transform="" R="255" G="2"/>
    <transform value=""/>
   </plot>
   <plot col="0" row="1">
    <range top="1.021419" right="99.000680" left="0.003578" bottom="-0.996958"/>
    <limitY/>
    <curve name="/xyz_debug_estimate/xy_plus/roll_bias" B="255" custom_transform="" R="0" G="36"/>
    <curve name="/xyz_debug_estimate/xy_plus/sigma_minus.3" B="0" custom_transform="" R="255" G="0"/>
    <curve name="/xyz_debug_estimate/xy_plus/sigma_plus.3" B="10" custom_transform="" R="255" G="0"/>
    <transform value=""/>
   </plot>
   <plot col="0" row="2">
    <range top="0.996605" right="99.000680" left="0.003578" bottom="-1.009491"/>
    <limitY/>
    <curve name="/xyz_debug_estimate/xy_plus/sigma_minus.4" B="0" custom_transform="" R="255" G="44"/>
    <curve name="/xyz_debug_estimate/xy_plus/sigma_plus.4" B="16" custom_transform="" R="255" G="0"/>
    <curve name="/xyz_debug_estimate/xy_plus/xa_bias" B="255" custom_transform="" R="0" G="4"/>
    <transform value=""/>
   </plot>
   <plot col="0" row="3">
    <range top="1.019746" right="99.000680" left="0.003578" bottom="-0.996892"/>
    <limitY/>
    <curve name="/xyz_debug_estimate/xy_plus/sigma_minus.5" B="3" custom_transform="" R="255" G="0"/>
    <curve name="/xyz_debug_estimate/xy_plus/sigma_plus.5" B="0" custom_transform="" R="255" G="13"/>
    <curve name="/xyz_debug_estimate/xy_plus/ya_bias" B="255" custom_transform="" R="0" G="0"/>
    <transform value=""/>
   </plot>
  </plotmatrix>
  <currentPlotMatrix index="1"/>
 </tabbed_widget>
 <use_relative_time_offset enabled="1"/>
 <Plugins>
  <DataLoad_CSV>
   <default time_axis=""/>
  </DataLoad_CSV>
  <DataLoad_ROS_bags>
   <selected_topics list=""/>
  </DataLoad_ROS_bags>
  <ROS_Topic_Streamer>
   <selected_topics list="/estimate_error;/rgbd_velocity/body_level_frame;/sync_estimates;/velocity/body_level_frame;/xyz_debug_estimate"/>
  </ROS_Topic_Streamer>
  <RosoutPublisherROS/>
  <TopicPublisherROS/>
 </Plugins>
</root>

